src/deserialize_sequence.cc

```diff
@@ -310,15 +312,15 @@ std::vector<Tag> parse_tags(gul14::string_view str)
     return tags;
 }
 
-bool parse_autorun(gul14::string_view str)
+bool parse_bool(gul14::string_view str)
 {
-    auto autorun{gul14::trim(str)};
-    if (autorun == "true" )
+    auto bool_expression{gul14::trim(str)};
```

Comment:
As a very minor optimization, this could be `trim_sv` so we do not have to copy the string.